Assets/Scripts/UI/ContextualHotKeysUI.cs

```diff
@@ -0,0 +1,81 @@
+using System.Collections;
+using System.Collections.Generic;
+using TMPro;
+using UnityEngine;
+using UnityEngine.UI;
+
+public class ContextualHotKeysUI : MonoBehaviour

```

Comment:
You need to set up an event so that the tooltip is updated if the user binds a new hotkey, so the UI updates

Reply:
Ok I did. let me know what you think.